src/agents/intern/__init__.py

```
@@ -44,6 +44,7 @@ def refresh_ticket_backlog(self):
         return len(next_tickets) == 0
 
     def refresh_pr_backlog(self):
+        return  # Not implemented yet
```

Let's raise an exception here

---

We can't, this function is being called by the loop so we leave it here as placeholder for future work right?